services/common/src/components/projectSummary/AuthorizationsInvolved.tsx

```diff
@@ -388,55 +325,52 @@ const RenderEMAAuthCodeFormSection = ({
       <Field
         name={`${code}.types`}
         component={RenderGroupCheckbox}
-        label="What type of authorization is involved in your application?"
         required
         validate={[requiredList]}
         normalize={normalizeGroupCheckBox}
+        onDrop={(event) => {
```

Comment:
This onDrop though was for a bug. I noticed that when I had "amendment" checked, and dropped a file, that it unchecked the box on me! Can be replicated in test, currently.